### PR TITLE
Add bridge onboarding service

### PR DESCRIPTION
**Pull request by LEAFERx**
# Goals of PR

Core changes:

- Add `packages/onboarding` which implements an API endpoint that helps finalize first-time user's deposit request.
- Improve `sdk`.
- Update lockfile.

### CODE REVIEW

File: .changeset/pink-chefs-change.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 "@specularl2/sdk": patch
 "@specularl2/contracts": patch
+"@specularl2/onbarding-services": patch
```
Comment (chalkandpaste) [inline .changeset/pink-chefs-change.md]: `onboarding-services`

Comment (LEAFERx) [reply .changeset/pink-chefs-change.md]: thanks, fixed

**Review by chalkandpaste — APPROVED** [review]
LGTM

**Review by siosw — APPROVED** [review]
looks good. don't think this is a big concern now but eventually I guess we'll need some sort of rate limit to avoid ppl draining the service

**Comment by LEAFERx** [comment]
> looks good. don't think this is a big concern now but eventually I guess we'll need some sort of rate limit to avoid ppl draining the service

they will cost more to initiate deposit on L1 side than our services's cost of finalizing them. But yes rate limit should be introduced

**Comment by LEAFERx** [comment]
A lot of `onboarding` code should be moved to `sdk` for general briding utilities.

**Pull request merged**